package.json

```diff
@@ -103,7 +103,10 @@
     "got": "^11.8.5",
     "loader-utils": "^2.0.4",
     "**/@storybook/**/minimatch": "^3.0.5",
-    "**/ember-electron/**/minimatch": "^3.0.5"
+    "**/ember-electron/**/minimatch": "^3.0.5",
+    "**/babel-core/json5": "^2.2.2",
```

Comment:
I think you only need this one to force the `^0.5.1` to the most recent version. According to your description, the other two actually have the correct semver needed to update to the most recent version. This can be done by manually removing the out of date entries in the `yarn.lock` and redoing an install to get the latest version that matches the semver. 

I suspect if you remove the two below and run `yarn install` again nothing will change since you have the correct `yarn.lock` file now.

Reply:
Ahh that makes so much more sense, thank you!

Reply:
Nicely executed @lisbet-alvarez !! As @ZedLi already mention, very clean the scope of the specific dep that needs to be resolved. Definetly a pattern to follow in the future!